docker/docker_client.go

```diff
@@ -264,15 +271,15 @@ func newDockerClient(sys *types.SystemContext, registry, reference string) (*doc
 	tr.TLSClientConfig.InsecureSkipVerify = skipVerify
```

Comment:
(Following up on 
> This does not need to exist both here and in `getOnesignature`; `detectProperties` can AFAICS create the `client` and store it in the `dockerClient` object.

apparently I can’t comment on that one now?!)

The point was not really to rename the field from `endpointSkipVerify`, rather than the `sys.DockerInsecureSkipTLSVerify` logic here is unnecessary (redundant/confusing) when `detectPropertiesHelper` does the same thing.  Drop the `DockerInsecureSkipTLSVerify` logic here, record `skipVerify` only based on `FindRegistry`, and rename back to `endpointSkipVerify` (or `endpointInsecureSkipTLSVerify`)

Reply:
I found out that we could get rid of an additional `skipVerify`/`endpointSkipVerify`/`endpointInsecureSkipTLSVerify` variable by simply using the new `tlsClientConfig.InsecureSkipVerify`. It makes the logic a bit easier to understand in my point of view. I'd chose to implement it that way.

Reply:
(It’s not trivially obvious this way that the `c.tlsClientConfig.InsecureSkipVerify` field is designed to be modified by callers, but, sure, less redundancy.)